examples/aks/aks_cluster/variables.tf

```diff
@@ -0,0 +1,35 @@
+# AKS  cluster variables.
+variable "cluster_name" {
+  type        = string
+  description = "Name of the AKS cluster, resources will be created for."
+}
+
+variable "cluster_region" {
+  type        = string
+  description = "Region of the AKS cluster, resources will be created for."
+}
+
+variable "cluster_version" {
+  type        = string
+  description = "AKS cluster version."
+  default     = "1.23"
+}
+
+
+# Variables required for connecting EKS cluster to CAST AI
+variable "castai_api_token" {
+  type = string
+  description = "CAST AI API token created in console.cast.ai API Access keys section"
+}
+
+variable "delete_nodes_on_disconnect" {
+  type        = bool
+  description = "Optional parameter, if set to true - CAST AI provisioned nodes will be deleted on cluster disconnection."
+  default     = true
```

Comment:
Not sure if this should be set to true. Is good for our testing etc but in production it should be set to `false`

Reply:
Yeah, that's true. However, I have set it to `true` for the case if the one would like to simply run Terraform apply on our examples in their cloud account and later run `terraform destroy`. The one would be sure that all infrastructure created by this example is removed. If leaving it to `false` I expect that not so advanced user could miss CAST AI created nodes and this would create bad experience for user for incurring cost which wasn't expected.

WDYT of leaving it `true` but update description that for Production use it is recommended to set it to false?

Reply:
I'm fine with adding the recommendation in description

Reply:
Updated as discussed.